### PR TITLE
CSI-1191_iscsi_login_deployment

### PR DESCRIPTION
**Pull request by igorgonibm**
Deploy iscsi login

### CODE REVIEW

File: pkg/controller/ibmblockcsi/syncer/csi_node.go [hunk 1]
```diff
@@ -299,6 +309,8 @@ func (s *csiNodeSyncer) ensureVolumes() []corev1.Volume {
 		ensureVolume("device-dir", ensureHostPathVolumeSource("/dev", "Directory")),
 		ensureVolume("sys-dir", ensureHostPathVolumeSource("/sys", "Directory")),
 		ensureVolume("host-dir", ensureHostPathVolumeSource("/", "Directory")),
+		ensureVolume("lib-modules", ensureHostPathVolumeSource("/lib/modules", "Directory")),
+		ensureVolume("iscsi", ensureHostPathVolumeSource("/etc/iscsi", "DirectoryOrCreate")),
```
Comment (oriyarde) [inline pkg/controller/ibmblockcsi/syncer/csi_node.go]: remove `OrCreate` (this was [never reviewed](https://github.com/IBM/ibm-block-csi-operator/pull/1))

File: pkg/controller/ibmblockcsi/syncer/csi_node.go [hunk 2]
```diff
@@ -144,6 +145,7 @@ func (s *csiNodeSyncer) ensureContainersSpec() []corev1.Container {
 		"SETGID",
 		"SETUID",
 		"DAC_OVERRIDE",
+		"SYS_ADMIN",
```
Comment (oriyarde) [inline pkg/controller/ibmblockcsi/syncer/csi_node.go]: Test if it works without it

File: pkg/controller/ibmblockcsi/syncer/csi_node.go [hunk 3]
```diff
@@ -97,6 +97,7 @@ func (s *csiNodeSyncer) ensurePodSpec() corev1.PodSpec {
 		Containers:         s.ensureContainersSpec(),
 		Volumes:            s.ensureVolumes(),
 		HostIPC:            true,
+		HostNetwork: 		true,
```
Comment (27149chen) [inline pkg/controller/ibmblockcsi/syncer/csi_node.go]: Is it mandatory? Please also consider the impact.

Comment (igorgonibm) [reply pkg/controller/ibmblockcsi/syncer/csi_node.go]: Doesn't work without it:
```bash-4.2# iscsiadm -m discoverydb -t st -p 172.18.2.13:3260 --discover
iscsiadm: can not connect to iSCSI daemon (111)!
iscsiadm: Cannot perform discovery. Initiatorname required.
iscsiadm: Could not perform SendTargets discovery: could not connect to iscsid```

**Comment by oriyarde** [comment]
@igorgonibm could we delete this branch?

**Pull request merged**